app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -641,6 +641,12 @@ ShortFormApplicationController = (
     link = $state.href('dahlia.listing', { id: $scope.listing.listingID })
     {listingLink: "<a href='#{link}'>#{linkText}</a>"}
 
+  $scope.listingIsRental = ->
```

Comment:
I'm curious to get @software-project and @callaghanc's feedback on including these functions since they've also been doing work on the application that involves checking if the listing isSale

Reply:
@akegan i'm down to try out using `isSale` vs `isRental` instead of `isSale` vs `!isSale`, perhaps we might find that it helps legibility

Reply:
Same here `isRental` makes sense.